xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAEBuffer.cpp

```diff
@@ -436,9 +436,10 @@ bool CActiveAEBufferPoolResample::ResampleBuffers(int64_t timestamp)
         if (!m_dspSample)
           m_dspSample = m_dspBuffer->GetFreeBuffer();
 
+        m_dspSample->timestamp = in->timestamp;
```

Comment:
looks like this code distinguishes between m_dspSample and ! m_dspSample
timestamp has to be set accordingly. If there are already samples in m_dspSample, this is not correct.